industrial_ci/scripts/run_travis

```diff
@@ -130,16 +130,19 @@ The jobs will be run in clean environments.
 Only DOCKER_PORT, SSH_AUTH_SOCK, and TERM will be kept.
 Additional variable names can be passed at the end.
 
-""" % cmd)
+"""
 
 def main(scripts_dir, argv):
-    if '--help' in argv:
-        print_help(argv[0])
-        sys.exit(0)
-
-    args, extra_env = parse_extra_args(argv[1:])
```

Comment:
Since we're dropping this `argv[1:]`, the 0th argument by default is the script's path itself, isn't it?

At least changing [this line](https://github.com/ros-industrial/industrial_ci/pull/300/files#diff-a4313c4772034419fabd7360eba4e6b3R40) to take 1st arg yields different result for me.

Reply:
Yes, this is needed.